boards/arm/mimx8mp_evk/mimx8mp_evk.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021, Laird Connectivity
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mimx8mp_evk
```

Comment:
Can you add a `supported` definition for UART, to enable the CI to test UART drivers?

Reply:
Done.